src/loader/endpoints/GoogleDocsLib.py

```diff
@@ -0,0 +1,46 @@
+import pickle
+from googleapiclient.http import MediaIoBaseDownload
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+import io
+import pandas as pd
+
+# Takes the id and token and reads the bytes of a file
+def getFileBytes(file_id, token_path="token.pickle"):
```

Comment:
Always write the function with lowercase and underscore.

for instance:
`get_file_bytes`